integration/check_test.go

```diff
@@ -84,7 +84,7 @@ func (s *SkopeoSuite) TestVersion(c *check.C) {
 func (s *SkopeoSuite) TestCanAuthToPrivateRegistryV2WithoutDockerCfg(c *check.C) {
 	out, err := exec.Command(skopeoBinary, "--docker-cfg=''", "--username="+s.regV2WithAuth.username, "--password="+s.regV2WithAuth.password, "inspect", fmt.Sprintf("docker://%s/busybox:latest", s.regV2WithAuth.url)).CombinedOutput()
 	c.Assert(err, check.NotNil, check.Commentf(string(out)))
-	wanted := "Error: image busybox not found"
+	wanted := "Invalid status code returned when fetching manifest 401"
 	if !strings.Contains(string(out), wanted) {
```

Comment:
Fine, I was saying that if you change this again in the future you need to change 4 lines.  As upposed to changing it one place in the test tool.  I don't think this should be external to the test module.